ghcide/src/Development/IDE/Plugin/HLS.hs

```diff
@@ -261,22 +276,25 @@ extensibleNotificationPlugins recorder xs = mempty { P.pluginHandlers = handlers
           Just fs -> do
             -- We run the notifications in order, so the core ghcide provider
             -- (which restarts the shake process) hopefully comes last
-            mapM_ (\(pid,_,f) -> otTracedProvider pid (fromString $ show m) $ f ide vfs params) fs
+            mapM_ (\(pid,_,f) -> otTracedProvider pid (fromString $ show m) $ f ide vfs params
+                                    `catchAny`
+                                    (\e -> logWith recorder Warning (ExceptionInPlugin pid (Some m) e))) fs
```

Comment:
Same question here, is this the `catchAny` from safe-exceptions?

Reply:
This is the catchAny from UnliftIO.Exception, and it doesn't catch async exceptions. It is the same one used for catching exceptions in normal plugin handlers, I am just adding it to the command and notification handlers here.